src/core/server/opensearch/legacy/api_types.ts

```diff
@@ -232,6 +232,7 @@ export interface LegacyAPICaller {
   (endpoint: 'cat.help', params: CatHelpParams, options?: LegacyCallAPIOptions): ReturnType<Client['cat']['help']>;
   (endpoint: 'cat.indices', params: CatIndicesParams, options?: LegacyCallAPIOptions): ReturnType<Client['cat']['indices']>;
   (endpoint: 'cat.master', params: CatCommonParams, options?: LegacyCallAPIOptions): ReturnType<Client['cat']['master']>;
+  (endpoint: 'cat.cluster_manager', params: CatCommonParams, options?: LegacyCallAPIOptions): ReturnType<Client['cat']['master']>;
```

Comment:
Does the return type not work for `Client['cat']['cluster_manager']`? This might require the bumping of the Node client `https://github.com/opensearch-project/opensearch-js/blob/e008fe82dc19b156d4eab8acf7022d36641878af/api/api/cat.js#L268`

Reply:
Added an issue here: https://github.com/opensearch-project/OpenSearch-Dashboards/issues/1818